src/catalogue/items/catalogueItemsLandingPage.component.tsx

```diff
@@ -554,26 +507,7 @@ function CatalogueItemsLandingPage() {
           </Grid>
         </Grid>
       )}
-      {!catalogueItemIdDataLoading ? (
-        !catalogueItemIdData && (
-          <Box
-            sx={{
-              width: '100%',
-              justifyContent: 'center',
-              marginTop: 1,
-            }}
-          >
-            <Typography sx={{ fontWeight: 'bold', textAlign: 'center' }}>
-              No result found
-            </Typography>
-            <Typography sx={{ textAlign: 'center' }}>
-              This catalogue item doesn&#39;t exist. Please click the Home
-              button on the top left of your screen to navigate to the catalogue
-              home.
-            </Typography>
-          </Box>
-        )
-      ) : (
+      {!(!catalogueItemIdDataLoading && !catalogueCategoryDataLoading) && (
```

Comment:
```suggestion
      {(catalogueItemIdDataLoading || catalogueCategoryDataLoading) && (
```
De Morgan's theorem, comes in handy very occasionally 😄